lib/tasks/sphinx.rake

```diff
@@ -2,6 +2,11 @@
 
 require 'progress_counter'
 
+# The queue that SphinxIndexationWorker normally uses is is 'default'
+# This rake task however enqueues into a different queue, which allows setting a lower
+# priority for this non-time-critical maintenance task, and even delete all enqueued jobs at once
+BULK_INDEXING_QUEUE = Rails.configuration.three_scale.bulk_indexing_queue || 'default'
+
```

Comment:
Delete this one? Other than that, LGTM. Please squash-merge this

Reply:
Thanks @akostadinov Your suggestions look good :+1: